src/interfaces/IPoolTogetherTimelock.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.16;
+
+interface IPoolTogetherTimelock {
+  event CancelTransaction(
+    bytes32 indexed txHash,
+    address indexed target,
+    uint256 value,
+    string signature,
+    bytes data,
+    uint256 eta
+  );
+  event ExecuteTransaction(
+    bytes32 indexed txHash,
+    address indexed target,
+    uint256 value,
+    string signature,
+    bytes data,
+    uint256 eta
+  );
+  event NewAdmin(address indexed newAdmin);
+  event NewDelay(uint256 indexed newDelay);
+  event NewPendingAdmin(address indexed newPendingAdmin);
+  event QueueTransaction(
+    bytes32 indexed txHash,
+    address indexed target,
+    uint256 value,
+    string signature,
+    bytes data,
+    uint256 eta
+  );
+
+  function acceptAdmin() external;
+  function admin() external view returns (address);
+  function cancelTransaction(
+    address target,
+    uint256 value,
+    string memory signature,
+    bytes memory data,
+    uint256 eta
+  ) external;
+  function delay() external view returns (uint256);
+  function executeTransaction(
+    address target,
+    uint256 value,
+    string memory signature,
+    bytes memory data,
+    uint256 eta
+  ) external payable returns (bytes memory);
+  function gracePeriod() external pure returns (uint256);
+  function maximumDelay() external pure returns (uint256);
+  function minimumDelay() external pure returns (uint256);
+  function pendingAdmin() external view returns (address);
+  function queueTransaction(
+    address target,
+    uint256 value,
+    string memory signature,
+    bytes memory data,
+    uint256 eta
+  ) external returns (bytes32);
+  function queuedTransactions(bytes32) external view returns (bool);
+  function setDelay(uint256 delay_) external;
+  function setPendingAdmin(address pendingAdmin_) external;
```

Comment:
Is this interface too broad? Should we pair back to a minimum number of functions necessary for the Governor?

Reply:
Yea I think is ok, I'll resolve this